Sources/Turf/Geometries/LineString.swift

```diff
@@ -215,4 +215,132 @@ extension LineString {
         
         return closestCoordinate
     }
+
+    private func squareDistance(from: CLLocationCoordinate2D, to: CLLocationCoordinate2D) -> Double {
```

Comment:
This is fine for a private method, but FYI it’s more idiomatic to use labels for prepositions, like this:

```swift
private func squareDistance(from origin: CLLocationCoordinate2D, to destination: CLLocationCoordinate2D) -> Double {
```

Reply:
Updated since your format is better.